### PR TITLE
Ubuntu fixes

### PR DESCRIPTION
**Pull request by pascalfleury**
This contains a few fixes for Ubuntu-based systems:
 - acprep decodes and compares strings (returned from the commands are bytes), which also works with Python3
 - updated the deps for bionic (18.04) which is the current LTS (Long Term Support) version
 - adjusted README.md to reflect these newer systems. Also removed instruction for 12.04 (which has not gotten any update since April 28, 2017)

### CODE REVIEW

File: acprep [hunk 1]
```diff
@@ -510,7 +500,7 @@ class PrepareBuild(CommandLineApp):
 
         self.log.info("Installing Ledger's build dependencies ...")
 
-        system = str(self.get_stdout('uname', '-s'))
+        system = self.get_stdout('uname', '-s').decode('utf8')
```
Comment (tbm) [inline acprep]: It seems you're undoing the change from the other commit there?

Comment (pascalfleury) [reply acprep]: yes that was a mistake, and I did not clean the change history.

**Comment by pascalfleury** [comment]
I am a bit lost as to how I can a) bring my fork up-to-date, and b) that the pull from ledger/ledger does not show up as a commit in my pull request for the deps changes in acprep.

It seems that syncing both github repos leavves too many traces...

**Comment by pascalfleury** [comment]
Yes, had no time to act on it yet.

I did not find a way to not have some of the commits show up in the PR, so
I created a new one (#1733).
Hopefully that one is clean and working.

Pascal

**Pull request closed without merging**